app/policies/proposal_policy.rb

```diff
@@ -20,8 +20,7 @@ def can_show!
   end
 
   def can_create!
-    # TODO restrict by client_slug
-    true
+    slug_matches? || @user.admin?
```

Comment:
It would be false in the following cases, most (but not all) of which have tests.

The user does not have a 'admin' role and one of the following is true:

* the user is missing a client_slug
* the user requests /ncr/* endpoint but has gsa18f client_slug
* the user requests /gsa18f/* endpoint but has ncr client_slug